deploy-manage/deploy/cloud-on-k8s/pod-disruption-budget.md

```diff
@@ -12,9 +12,45 @@ products:
 
 A [Pod Disruption Budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) (PDB) allows you to limit the disruption to your application when its pods need to be rescheduled for some reason such as upgrades or routine maintenance work on the Kubernetes nodes.
 
-ECK manages a default PDB per {{es}} resource. It allows one {{es}} Pod to be taken down, as long as the cluster has a `green` health. Single-node clusters are not considered highly available and can always be disrupted.
+ECK manages either a single default PDB, or multiple PDBs per {{es}} resource according to the license available.
```

Comment:
```suggestion
ECK manages either a single PDB, or multiple PDBs per {{es}} resource depending on the license installed.
```
Not sure if it is better.
Other ideas: 

"depending on the license level of the ECK installation" (because technically users upgrade ECK to Enterprise not individual ES clusters)